client/src/components/App.jsx

```diff
@@ -1,34 +1,48 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import FeaturesDetails from './FeaturesDetails';
+import axios from 'axios';
+// import PropTypes from 'prop-types';
+import style from './styles.css';
+import FeaturesDetails from './FeaturesDetails/FeaturesDetails';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      features: [{ value: '1' }, { value: '2' }, { value: '3' }],
+      header: '',
+      features: [],
+      activeIndex: 0,
+
     };
+    this.getFeatures = this.getFeatures.bind(this);
+  }
+
+  componentDidMount() {
+    this.getFeatures(1);
+  }
+
+  getFeatures(id) {
+    axios.get(`http://localhost:8080/products/${id}`)
+      .then((response) => {
+        console.log(response);
+        this.setState({
```

Comment:
I believe airbnb style guide wants us to use callback in setState.
Would you mind trying the following code?

this.setState({() =>header: response.data.product_features.header }); 

I wonder whether you got a warning for the setState method from ESlint.
If I am wrong, please let me know :)